exonum-java-binding/core/rust/src/runtime/config.rs

```diff
@@ -69,6 +75,15 @@ pub struct InternalConfig {
     pub system_lib_path: String,
 }
 
+impl Default for InternalConfig {
+    fn default() -> Self {
+        Self {
+            system_class_path: system_classpath(),
```

Comment:
Is it reasonable to search the file system in the implementation of Default trait?

> A trait for giving a type a useful default value.
Sometimes, you want to fall back to some kind of default value, and don't particularly care what it is

Reply:
Given that we use `default` internal config in EJB App and `non-default` in tests, I think it might be possible. But I agree that it is too subtle, so replaced with `app_config` method.